### PR TITLE
disfetch: init at 1.12

### PR DESCRIPTION
**Pull request by q60**
###### Motivation for this change
Disfetch is another *nix distro fetching program, but less complex. It is based on several principles
+ less code, more efficiency, more readability
+ simplicity
+ logos must be only 8 rows
+ using only coreutils, other built-ins, bash and nothing third-party
+ showing only main and needed system information

https://github.com/llathasa-veleth/disfetch

I am sorry for messing pull request up with force push.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/disfetch/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  installPhase = ''
+    install -Dm755 -t $out/bin disfetch
+  '';
+
+  meta = with lib; {
+    description = "Yet another *nix distro fetching program, but less complex";
+    homepage = "https://github.com/llathasa-veleth/disfetch";
+    license = licenses.mit;
+    platforms = platforms.all;
+    maintainers = [ maintainers.v ];
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/disfetch/default.nix:27]:
```suggestion
    maintainers = [ maintainers.vel ];
```

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107811` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>disfetch</li>
  </ul>
</details>

**Comment by llathasa-veleth** [comment]
@SuperSandro2000 hey? i have done everything that was needed

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review. If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 107811` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>disfetch</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107811` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>disfetch</li>
  </ul>
</details>

**Pull request merged**